std_msgs/Header header
  uint32 seq
  time stamp
  string frame_id

geometry_msgs/Point timeMotion
geometry_msgs/Point point
geometry_msgs/Point pointVel
geometry_msgs/Point pointAcc
geometry_msgs/Point pointJerk
